core/src/test/scala/sttp/client3/RequestTests.scala

```diff
@@ -65,4 +65,37 @@ class RequestTests extends AnyFlatSpec with Matchers {
       .show() shouldBe
       "response as: (either(as string, as params), as string), headers: Accept-Encoding: gzip, deflate, Authorization: ***, Content-Type: text/plain; charset=utf-8, Content-Length: 4, body: string: 1234"
   }
+
```

Comment:
Thanks for adding those, I wasn't able to run the testsuite myself, not sure why sbt was complying.